tequilapi/endpoints/stop.go

```diff
@@ -0,0 +1,20 @@
+package endpoints
+
+import (
+	"github.com/julienschmidt/httprouter"
+	"net/http"
+)
+
+// Stopper stops application and performs required cleanup tasks
+type Stopper func()
```

Comment:
Is duplicated interface `Stopper` needed in both packages?

Reply:
Yes - interfaces are defined on receiver side and they are very lightweight, server the purpose of labeling what these functions expect.